src/Shell/Task/ModelTask.php

```diff
@@ -569,6 +571,17 @@ public function fieldValidation($fieldName, array $metaData, $primaryKey)
             ]
         ];
 
+        foreach ($schema->constraints() as $constraint) {
+            $constraint = $schema->constraint($constraint);
+            if (!in_array($fieldName, $constraint['columns']) || count($constraint['columns']) > 1) {
+                continue;
+            }
+
```

Comment:
I does work with multiple columns, but extra columns need to be passed as `scope`